mllib/src/test/scala/org/apache/spark/ml/feature/CountVectorizerSuite.scala

```diff
@@ -183,6 +183,26 @@ class CountVectorizerSuite extends SparkFunSuite with MLlibTestSparkContext
       case Row(features: Vector, expected: Vector) =>
         assert(features ~== expected absTol 1e-14)
     }
+
+    // CountVectorizer test
+    val df2 = sqlContext.createDataFrame(Seq(
+      (0, split("a b c d e a b"),
+        Vectors.sparse(5, Seq((0, 1.0), (1, 1.0), (2, 1.0), (3, 1.0), (4, 1.0)))),
+      (1, split("a a a a a a"), Vectors.sparse(5, Seq((0, 1.0)))),
+      (2, split("c c"), Vectors.sparse(5, Seq((2, 1.0)))),
+      (3, split("b b b b b"), Vectors.sparse(5, Seq((1, 1.0)))))
+    ).toDF("id", "words", "expected")
+    val cv2 = new CountVectorizer()
+      .setInputCol("words")
+      .setOutputCol("features")
+      .setBinary(true)
+      .fit(df2)
+    assert(cv2.vocabulary === Array("a", "b", "c", "d", "e"))
```

Comment:
I put there to catch the possible vocabulary index change. I will remove it. 
Thanks!
Miao